jobs/acceptance-tests/spec

```diff
@@ -78,3 +78,7 @@ properties:
     description: Timeout for long curls
   acceptance_tests.broker_start_timeout:
     description: Timeout for broker starts
+  acceptance_tests.existing_user:
+    description: The username of an existing regular user.
+  acceptance_tests.existing_user_password:
+    description: The password of an existing regular user.
```

Comment:
Suggested change:

"The password of the existing user.  Only required if the existing user property is also being set."